config/this-week-in-open-source.config.json

```
@@ -26,7 +26,10 @@
         "BobrImperator/tilex",
         "emberfest/emberfest.github.io",
         "mansona/ember-paper",
-        "mansona/rfcs"
+        "mansona/rfcs",
+        "fzerocentral/fzerocentral-web",
+        "hugopeixoto/pokemon-collection-tracker-frontend",
+        "test-renovate-repo"
```

Remove `test-renovate-repo`

```suggestion
        "hugopeixoto/pokemon-collection-tracker-frontend"
```